examples/resources/mondoo_integration_slack/resource.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
```

Comment:
I recommend moving the `terraform` section out to `main.tf`. This way it does not show up in the rendered docs. See https://github.com/mondoohq/terraform-provider-mondoo/blob/main/examples/resources/mondoo_integration_azure/main.tf.

Tflint also requires us to add version constraints like:

```hcl
mondoo = {
      source  = "mondoohq/mondoo"
      version = ">= 0.4.0"
}
```